sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceManagementClient.cs

```diff
@@ -3679,7 +3679,7 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> ImportDevicesAsync(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
+        public virtual async Task<Operation> ImportDevicesAsync(WaitUntil waitUntil, string action, RequestContent content, RequestContext context = null)
```

Comment:
@mikekistler, can you confirm that there is no value returned from the `ImportDevices` LRO?

Reply:
I cannot confirm what is returned from this operation. I had hoped to create a test environment to determine this empirically but it proved too complicated.

Reply:
@dpokluda, could you confirm that there is no value returned from the ImportDevices LRO?